src/main/java/com/minecolonies/coremod/colony/requestsystem/management/manager/StandardRequestManager.java

```diff
@@ -175,12 +175,13 @@ private void markDirty()
     @Override
     public boolean isDirty()
     {
-        if (dirty)
```

Comment:
This is a half assed hotfix(?!??) for something without description. And I am 100% sure this breaks a lot of things. Please explain why this change is needed and how it prevents your not described actual concrete problem you are trying to fix!